catboost/python-package/catboost/_catboost.pyx

```diff
@@ -3854,6 +3857,21 @@ cdef class _PoolBase:
         )
         self.target_type = pool.target_type
 
+
+    cpdef save_quantization_borders(self, output_file):
```

Comment:
Let's add a documentation comment.
It is preferable because it is better to mention format of the borders file and give a link to the documentation.